src/engine/openspaceengine.cpp

```diff
@@ -470,16 +471,22 @@ void OpenSpaceEngine::initializeGL() {
 
     loadFonts();
 
+    constexpr std::chrono::seconds ScreenLogTimeToLive(20);
+    std::unique_ptr<ScreenLog> log = std::make_unique<ScreenLog>(
+        ScreenLogTimeToLive,
+        ScreenLog::LogLevel::Warning
+    );
+    ScreenLog* loadScreenLogger = log.get();
+    ghoul::logging::LogManager::ref().addLog(std::move(log));
+
```

Comment:
I think it would make more sense for the `LoadingScreen` class to own its ScreenLog?  Is there a reason why the OpenSpaceEngine would need to own it?